spec/dummy/app/assets/stylesheets/bootstrap_and_overrides.scss

```diff
@@ -0,0 +1,7 @@
+html {
+  font-size: 14px;
+}
+
+// Custom bootstrap variables must be set or imported *before* bootstrap.
+$secondary: #f5f5f5;
```

Comment:
Why override the secondary button color, if we use btn-light it looks like the same as this overridden one.

Reply:
It does look very similar. I decided to not override for now